minimesos/src/main/java/com/containersol/minimesos/mesos/Consul.java

```diff
@@ -0,0 +1,40 @@
+package com.containersol.minimesos.mesos;
+
+import com.containersol.minimesos.container.AbstractContainer;
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.model.ExposedPort;
+
+import java.util.TreeMap;
+
+/**
+ * Base, unmolested Mesos master class
+ */
+public class Consul extends AbstractContainer {
+
+    public static final String CONSUL_IMAGE_NAME = "containersol/consul-server";
+    public static final String CONSUL_TAG_NAME = "0.6";
```

Comment:
I think it'd be nice to make the version configurable. I'll send a pull request for that in a bit.